packages/client/manifest.json

```diff
@@ -2,7 +2,9 @@
   "features": {
     "spectrumThemes": true,
     "intelligentLoading": true,
-    "deviceAwareness": true
+    "deviceAwareness": true,
+    "state": true,
```

Comment:
Out of curiosity, how exactly does this work? We attach certain features to client lib versions so that we know if they are compatible or not when you upgrade an old app?

Reply:
Yep exactly! So I introduced this concept a while back to be able to know (in the builder) if the current app supports certain features. A lot of features involve a configuration part in the builder (which is always the latest version) and the corresponding implementation in the client lib (which is versioned). So these flags are used by the builder to know whether to show or hide stuff.
With this feature, the new flag is used to know whether or not to show the full theme editor (in updated apps) or just show the old theme dropdown (in apps that haven't been updated yet).
I also snuck a feature flag in for the recently merged state feature, because I realised that I hadn't put in a flag for that - so you could configure state bindings in the builder but they would never show up in the app.